src/libsync/configfile.h

```diff
@@ -209,6 +209,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
 
 private:
     typedef QSharedPointer<AbstractCredentials> SharedCreds;
+    const std::chrono::seconds DefaultRemotePollInterval { 30 }; // default remote poll time in milliseconds
```

Comment:
I'd kept that in the .cpp but I guess I won't be able to motivate you :D

Reply:
Sure u a.